blocks/utils.py

```diff
@@ -71,9 +71,43 @@ def unpack(arg, singleton=False):
 
 
 def shared_floatx_zeros(shape, **kwargs):
+    """Creates a shared variable array filled with zeros.
```

Comment:
If the docstring contains backslashes, you'll need to make the string raw. This just means it needs to start with `r"""` instead of `"""`